pkg/apis/rbac/validation/rulevalidation.go

```diff
@@ -40,7 +39,7 @@ type AuthorizationRuleResolver interface {
 }
 
 // ConfirmNoEscalation determines if the roles for a given user in a given namespace encompass the provided role.
-func ConfirmNoEscalation(ctx api.Context, ruleResolver AuthorizationRuleResolver, rules []rbac.PolicyRule) error {
+func ConfirmNoEscalation(ctx api.Context, ruleResolver AuthorizationRuleResolver, rules []rbac.PolicyRule) bool {
```

Comment:
how is this related to the authorizer interface change? I'd rather preserve the information with the error return